lib/worker-type.js

```diff
@@ -719,7 +719,10 @@ WorkerType.prototype.determineSpotBids = function (managedRegions, pricing, chan
         for (let zone of zones) {
           try {
             let potentialBid = pricingData[region][type][zone];
-            let potentialPrice = potentialBid / uf[type];
+            let bias = biaser.getBias(region, zone, type);
+            let potentialPrice = potentialBid / uf[type] * bias;
```

Comment:
nit, and this might just be me who is bad at math... but should we maybe add a parenthesis, so it's clear what order operations are applied...